Themes/default/languages/ManageSettings.english.php

```diff
@@ -56,6 +56,11 @@
 $txt['invalid_cookie_domain'] = 'The domain introduced seems to be invalid, please check it and save again.';
 $txt['secureCookies'] = 'Force cookies to be secure';
 $txt['httponlyCookies'] = 'Force cookies to be made accessible only through the HTTP protocol';
+$txt['samesiteCookies'] = 'Force cookies to be sent only to first parties';
+$txt['samesiteNone'] = 'None';
+$txt['samesiteLax'] = 'Lax';
+$txt['samesiteStrict'] = 'Strict';
+$txt['samesiteSecureRequired'] = 'If Same Site cookies is set to none, cookies must be secure.';
```

Comment:
Replace "is set" with "are set".

Reply:
It's a single setting, though, not multiple.  Reworded & capitalized to make it clear we are talking about a specific setting & option.